lib/rex/socket/range_walker.rb

```diff
@@ -23,6 +23,9 @@ module Socket
 ###
 class RangeWalker
 
+  MATCH_IPV4_RANGE = /^([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})-([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})$/
```

Comment:
My thought was not right now since it's not used elsewhere. Also it was my impression that the `MATCH_IPV4` constant shouldn't be used and that the `#ip_v4?` function should be used instead. Because of this, moving it with the other constants without writing a function might suggest to future readers that it should be used directly which would be inconsistent with the patterns I've seen.